src/dotnet/APIView/APIView/Model/TokenTreeModel.cs

```diff
@@ -0,0 +1,235 @@
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace APIView.TreeToken
+{
+    public enum StructuredTokenKind
+    {
+        Content = 0,
+        LineBreak = 1,
+        NoneBreakingSpace = 2,
```

Comment:
Travis pointed this out somewhere in his comments. The correct name should be `NonBreakingSpace`